cmake/utils.cmake

```diff
@@ -68,7 +68,7 @@ function(mayaUsd_find_python_module module)
             set(${module}_FIND_REQUIRED TRUE)
         endif()
         execute_process(COMMAND "${Python_EXECUTABLE}" "-c"
-            "import re, ${module}; print re.compile('/__init__.py.*').sub('',${module}.__file__)"
+            "from __future__ import print_function; import re, ${module}; print(re.compile('/__init__.py.*').sub('',${module}.__file__))"
```

Comment:
The python code is used to determine if a given module (such as jinja2) exists. It needed to be made python3 compatible.